builder/amazon/common/ami_config_test.go

```diff
@@ -29,26 +34,52 @@ func TestAMIConfigPrepare_name(t *testing.T) {
 	}
 }
 
+type mockEC2Client struct {
+	ec2iface.EC2API
+}
+
+func (m *mockEC2Client) DescribeRegions(*ec2.DescribeRegionsInput) (*ec2.DescribeRegionsOutput, error) {
+	// Describes a region.
+	regionString := "us-east-1"
+	regEast1 := ec2.Region{RegionName: &regionString}
```

Comment:
You could use: `ec2.Region{RegionName: aws.String("us-east-1"}` here to reduce the number of names in scope.

Reply:
Aha, good point. I'll clean it up.

Reply:
yeah, this is much better. Thanks for pointing it out